storj-exporter.py

```diff
@@ -18,21 +18,25 @@ def get_data(self):
     return self.call_api("sno/")
 
   def get_satellites(self):
-    satellites = []
+    satellites = {}
     for item in self.data['satellites']:
-      satellites.append(item['id'])
+      satellites.update({item['id']: item})
```

Comment:
I think it's better to directly use self.data['satellites'] object's array instead of a new dict